### DISCUSSION TITLE
uninstall stopped service

### DISCUSSION THREAD

**Discussion by geraldFS** [music-assistant/support#1134] [Q&A] [answered]
Hello,

after mass hasn't worked for a while and the service isn't loaded, I wanted to remove it.

Unfortunately this does not work.

How can you remove the service under integrations?
Manuel?

Home Assistant 2023.3.3
Supervisor 2023.03.1
OS 9.5
Front end 20230309.0 - latest

Greetings
gerald

**Comment by heinzma** [comment, marked as answer]
try this: https://github.com/music-assistant/hass-music-assistant/issues/1106#issuecomment-1451540503
